anonymize.py

```diff
@@ -1,11 +1,49 @@
-from PyPDF2 import PdfReader
+import json
```

Comment:
you don't use this library, please remove it

Reply:
weird this doesn't appear in my code or the code in the repo!

Reply:
you mean locally? if you want to show the difference between the last commit and your current code, you can run `git diff`